### ISSUE TITLE
Boolean Options are not handled correctly

### ISSUE THREAD

**Issue by aajtodd** [medmunds/aws-cfn-ses-domain#10]
## Summary
The lambda function does not handle `EnableSend` or `EnableReceive` properly.

The `true`/`false` values are converted to textual `"true"`/`"false"` not actual Python boolean values. 

You can see this in the log output of the lambda function:

```
Expanded properties to {'Domain': 'example.com, 'EnableSend': 'false', 'EnableReceive': 'true', 'MailFromSubdomain': 'mail', 'CustomDMARC': '', 'TTL': '1800', 'Region': 'us-east-1'}
```
Note I removed the service token and obviously changed the domain name in the output.

## Steps to reproduce

1. Create `test.cf.yaml`:
```yaml
  SESDomain:
    Type: Custom::SES_Domain
    Properties:
      ServiceToken: !GetAtt CfnSESDomain.Outputs.Arn
      Domain: "example.com"
      EnableSend: false
      EnableReceive: true
      MailFromSubdomain: "mail"
      TTL: "1800"
      CustomDMARC: ''
      Region: !Ref "AWS::Region"
 ```

2. Run `aws cloudformation deploy --capabilities CAPABILITY_IAM --template-file test.cf.yaml ...`

3. Check stack status in CloudFormation dashboard
   
### Results
SES outbound sending was enabled.

### Expected results
SES outbound sending should not be enabled.

## System
* aws-cfn-ses-domain version: 0.2
* aws cli version: aws-cli/1.16.159 Python/3.7.3 Darwin/17.7.0 botocore/1.12.155
* OS: Mac

## Additional Info

The unit tests seem to cover this case but they use Python boolean values in the parameters which is not how the parameters are actually passed to the function at runtime.

Also thanks, nice library overall.

**Comment by medmunds** [comment]
Thanks for the detailed report and for including the lambda log that demonstrates exactly what's going wrong.

Apparently CloudFormation doesn't use standard YAML or JSON parsing for templates. (Both YAML and JSON treat `true` and `false` as boolean literals.) And apparently this is a long-[known issue](https://forums.aws.amazon.com/thread.jspa?threadID=302268&tstart=0&messageID=900399#900399), though not yet mentioned in the CFN custom resource [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/crpg-ref-requests.html).

I guess I'll add some string-to-boolean conversion. In the meantime, a workaround should be to use zero instead of false: `EnableSend: 0`. (Or an empty string: `EnableSend: ""`.)

**Comment by medmunds** [comment]
Fix released in v0.3
